bindings/rust/s2n-tls/src/config.rs

```diff
@@ -595,6 +631,76 @@ impl Builder {
         Ok(self)
     }
 
+    /// Enable negotiating session tickets in a TLS connection
+    pub fn enable_session_tickets(&mut self, enable: bool) -> Result<&mut Self, Error> {
+        unsafe {
+            s2n_config_set_session_tickets_onoff(self.as_mut_ptr(), enable.into()).into_result()
+        }?;
+        Ok(self)
+    }
+
+    /// Adds a key which will be used to encrypt and decrypt session tickets. The intro_time parameter is time since
+    /// the Unix epoch (Midnight, January 1st, 1970). If this is 0, then intro_time is set to now.
+    pub fn add_session_ticket_key(
+        &mut self,
+        key_name: &[u8],
+        key: &[u8],
+        intro_time: Duration,
```

Comment:
Seems better to use [SystemTime](https://doc.rust-lang.org/std/time/struct.SystemTime.html) for this